azure/activity_logs_monitoring/index.js

```diff
@@ -284,12 +284,25 @@ class EventhubLogForwarder {
         return this.addTagsToJsonLog(jsonLog);
     }
 
-    formatSourceType(sourceType) {
-        if (sourceType.includes('microsoft.')) {
-            return sourceType.replace('microsoft.', 'azure.');
-        } else {
-            return '';
+    createResourceIdArray(record) {
+        // Convert the resource ID in the record to an array, handling beginning/ending slashes
+        var resourceId = record.resourceId.toLowerCase().split('/');
+        if (resourceId[0] === '') {
+            resourceId = resourceId.slice(1);
+        }
+        if (resourceId[resourceId.length - 1] === '') {
+            resourceId.pop();
         }
+        return resourceId;
+    }
+
+    isSource(resourceIdPart) {
+        // Determine if a section of a resource ID counts as a "source," in our case it means it starts with 'microsoft.'
+        return resourceIdPart.startsWith('microsoft.') ? true : false;
```

Comment:
This could just be `return resourceIdPart.startsWith('microsoft.');`

Reply:
Wow good catch idk what I was thinking, I'll update